src/modules/images/useImages.js

```diff
@@ -150,12 +151,26 @@ const useImages = () => {
     return Platform.OS === 'ios' ? imageURI : 'file://' + imageURI;
   };
 
+  const resizeImageIfNecessary = async (imageData) => {
+    let height = imageData.height;
+    let width = imageData.width;
+    const tempImageURI = Platform.OS === 'ios' ? imageData.uri || imageData.path : imageData.uri || 'file://' + imageData.path;
+    if (!height || !width) ({height, width} = await getImageHeightAndWidth(tempImageURI));
+    let resizedImage = {};
+    if (height > (deviceDimensions.width * 4) || width > (deviceDimensions.width * 4)) {
```

Comment:
deviceDimensions.width is here twice.  The first arg should be deviceDimensions.height